### ISSUE TITLE
Ensure all map components are labelled

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#937]
**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/3-a-1/

**Screen shot:**
![image](https://user-images.githubusercontent.com/43404358/94693977-42da9e00-032c-11eb-97c5-f86b2c99854f.png)

The slider is a custom component with no role or programmatically associated value. It is not keyboard focusable and it has no label to identify it.

Additionally, many of the links do not have a label to allow them to be accessed by voice activation software users. The purpose of these links is indicated visually by an icon with the text label available via a tooltip which appears on mouse hover.
Some of the links on the page have a role of button (zoom in, zoom out and home), the rest do not. This is inconsistent.

**Solution:**
Please ensure that all components are labelled.

**Example:**
`<a class="leaflet-control-fullscreen-button leaflet-bar-part" href="#" title="View Fullscreen" style="outline: none;" role="button" aria-label="View fullscreen"></a>`

The slider needs to be implemented as a focusable component with its functionality made accessible via the keyboard such as the cursor keys. Its behaviour should be described to screen reader users. It is advised to use native HTML components where possible to ensure that assistive technology can engage with it as expected such as the slider role, or the input type range component. For more information, please visit [Horizontal Slider Example | WAI-ARIA Authoring Practices 1.1](https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-1.html).

**Comment by brockfanning** [comment]
These are all third-party libraries, but I'll look into ways to fix the problems.

**Comment by LucyGwilliamAdmin** [comment]
I checked this and looks good to me: http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/map-slider-accessibility

However I've sent DAC feature branches for all the fixes so far - do you want me to wait for their feedback to approve?

**Comment by brockfanning** [comment]
I'm ok either way - just to make clear though, #941 only fixes part of this issue. I still need to fix the search and fullscreen buttons - I think they'll each be in their own PRs.

**Comment by LucyGwilliamAdmin** [comment]
Closing as fixed
